linera-chain/src/chain.rs

```diff
@@ -507,9 +508,9 @@ where
 
         // Update the (regular) outboxes.
         let outbox_counters = self.outbox_counters.get_mut();
-        for recipient in recipients {
-            let target = Target::chain(recipient);
-            let outbox = self.outboxes.load_entry_mut(&target).await?;
+        let targets: Vec<Target> = recipients.into_iter().map(Target::chain).collect();
```

Comment:
nit: The tradition is to minimize the type hint for `collect`, that is: `Vec<_>`.

Personally I don't having the type hint on this side of the definition and write `let targets = ... .collect::<Vec<_>>()`.

Reply:
Ok